### ISSUE TITLE
Extensions Taken Down by Web Store

### ISSUE THREAD

**Issue by wjywbs** [wjywbs/chrome-newtab-reloaded#17]
Both New Tab Reloaded and New Tab Reloaded (Extended) were taken down recently by the web store (2014/11/19 3:50 and 2014/11/18 1:27, GMT -5).

This is the email:

> Your Google Chrome item, "New Tab Reloaded," with ID: jliilhbdldnjbdbpajaakhpjpahnopbn did not comply with our policies and was removed from the Google Chrome Web Store.
> 
> Your item did not comply with the following section of our policy:
> 
> Do not use irrelevant, misleading, or excessive keywords in app descriptions, titles, or metadata.
> 
> If you'd like to re-submit your item, please make the appropriate changes to the item so that it complies with our policies, then re-publish it in your developer dashboard. Please reply to this email for issues regarding this item removal.
> 
> *Please keep in mind that your re-submitted item will not be immediately published live in the store. All re-submitted items undergo a strict compliance review and will be re-published if the item passes review.
> ......

I'm going to simplify the description to "Restore the previous new tab page for Chrome 33+." Considering the amount of users of the extended version, I will republish it as well.

Any ideas about how to "fix" this?

**Comment by kkamperschroer** [comment]
Have you had any additional issues since? I own an extension and received such a notice this morning after four years without issue. I have no idea why it was pulled, but your comments here are helpful and give me some things to try.

**Comment by wjywbs** [comment]
It depends on the reason you received. I received "do not use irrelevant, misleading, or excessive keywords", so I removed and rephrased some sentences in the description and republished.

For example, I removed these lines:

Since Chrome 33, Instant Extended API was removed from about:flags. It means there's no settings to disable this new feature. This extension overrides the "New Tab" page with the previous one with modifications to work with the new NTP "New Tab" page environment. Still, go to the Google Chrome support forum and tell Google to add the previous feature back.

Version 0.1.1: This extension was taken down by the web store due to "irrelevant, misleading, or excessive keywords in app descriptions, titles, or metadata". I rephrased some sentences.

Keyword:
Old New Tab Page, Classic New Tab Page, Instant Extended API Removed from Flags, Restore Old Behavior of New Tab Page, instant-extended-api flag
